libpromises/known_dirs.c

```diff
@@ -104,6 +107,21 @@ const char *GetWorkDir(void)
     return workdir == NULL ? GetDefaultWorkDir() : workdir;
 }
 
+const char *GetBinDir(void)
+{
+    const char *workdir = getenv("CFENGINE_TEST_OVERRIDE_WORKDIR");
+
+    if (workdir == NULL)
+    {
+        return GetDefaultBinDir();
```

Comment:
I'm not good at C, but I don't see this function defined anywhere

Reply:
It's defined by a weird macro in that file. Took me a while to figure that out too and I consider myself quite good at C. :smile: